linux-anvil-aarch64/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN localedef -i en_US -f UTF-8 en_US.UTF-8
 RUN yum update -y && \
     yum install -y \
         bzip2 \
+        findutils \
```

Comment:
`findutils` is already installed, so this is a no-op. However, I like to be explicit about what we use here.

Reply:
Sure, but I want to keep this list as small as possible